packages/chatbot-server-mongodb-public/src/tracing/postCommentToSlack.ts

```diff
@@ -149,22 +149,31 @@ ${
     : "No LLM-as-Judge Scores"
 }`;
 
+  const messageBlocks = [
+    Blocks.Section({ text: feedbackMd }),
+    Blocks.Divider(),
+    Blocks.Section({ text: messagesMd }),
+    Blocks.Section({ text: referencesMd }),
+    Blocks.Divider(),
+    Blocks.Section({ text: scoresMd }),
+    Blocks.Divider(),
+    Blocks.Section({ text: messageMetadataMd }),
+    Blocks.Divider(),
+    Blocks.Section({ text: idMetadataMd }),
+  ];
+
+  const maybeAuthUser = conversation.customData?.authUser;
+  if (maybeAuthUser && typeof maybeAuthUser === "string") {
+    messageBlocks.unshift(
+      Blocks.Section({ text: `Auth User: ${Md.user(maybeAuthUser)}` })
+    );
+  }
```

Comment:
This looks pretty similar to the work we do in `routesUpdateTraceHandlers` - should we pass in `authUser` from there as an arg to this function instead of recomputing?

Reply:
no i dont think so b/c we don't have a clean way to pass w/o major refactors